tests/app/views/test_brief_response.py

```diff
@@ -449,37 +525,20 @@ def test_can_not_update_brief_response_if_supplier_is_ineligible_for_brief(self,
             assert data == {'error': 'Supplier is not eligible to apply to this brief'}
 
     def test_can_not_update_brief_response_that_has_already_been_submitted(self, live_dos_framework):
-            # Create dummy brief_response which will validate when submitted.
-            brief_response_id = self.setup_dummy_brief_response(
-                brief_id=self.brief_id,
-            )
-
-            # Update brief response
-            res = self._update_brief_response(brief_response_id, {'respondToEmailAddress': 'newemail@email.com'})
-            assert res.status_code == 400
-            data = json.loads(res.get_data(as_text=True))
-            assert data == {'error': 'Brief response must be a draft'}
-
-    def test_can_not_update_brief_response_with_invalid_content(self, live_dos_framework):
-        res = self._update_brief_response(
-            self.brief_response_id, {'essentialRequirements': [False, False, False, False]}
+        # Create dummy brief_response which has been submitted
+        brief_response_id = self.setup_dummy_brief_response(
```

Comment:
You can override this behaviour by calling `setup_dummy_brief_response(submitted_at=None)` so not sure if it makes sense to rename it (I'm 50 50 on that). 

We could maybe rename it to something better like `setup_brief_response_fixture` though? Although maybe the word "dummy" is useful as it implies you are using example brief response data rather than passing in brief response data?

Words are hard...